c2cpg/src/main/scala/io/shiftleft/c2cpg/astcreation/AstCreator.scala

```diff
@@ -106,3 +107,17 @@ class AstCreator(val filename: String, val global: Global, val config: C2Cpg.Con
   }
 
 }
+
+object AstCreator {
+
+  @nowarn
```

Comment:
```suggestion
  @nowarn
    /** The CDT utility method is unfortunately in a class that is marked as deprecated, however, 
    * this is because the CDT team would like to discourage its use but at the same time does 
    * not plan to remove this code.
    */
```